### ISSUE TITLE
Revit API Select Parameter Type Node Doubt

### ISSUE THREAD

**Issue by NonicaTeam** [DynamoDS/DynamoRevit#2773]

**Comment by NonicaTeam** [comment]
?

**Comment by ZiyunShang** [comment]
Hi @NonicaTeam , "Select Parameter Type" and "Select BuiltIn Parameter Group" will be removed in coming version, and they will be replaced by "Spec Types" and "Group Types" nodes.

**Comment by NonicaTeam** [comment]
Thanks @ZiyunShang  for your message. By coming version you mean Dynamo for Revit 2.13?

**Comment by ZiyunShang** [comment]
Yes @NonicaTeam , you may refer to this PR - https://github.com/DynamoDS/DynamoRevit/pull/2770

**Comment by johnpierson** [comment]
Closing this out - a fix for this was merged, as noted in the thread above. The issue was simply never closed.

If you still see this behaviour on a current build, please comment and we will reopen.
